### PR TITLE
chore: added useEventListener Hook and linting

### PR DESCRIPTION
**Pull request by anikethsaha**
Added the `useEventListener ` hook referenced from [`here`](https://codesandbox.io/s/z64on3ypm)

### CODE REVIEW

File: hooks.js [hunk 1]
```diff
@@ -151,5 +154,72 @@ export const hooks = [
     );
   }
     `
+  },
+  {
+    name: "useEventListener",
+    author: "Gabe Ragland",
+    link: "https://codesandbox.io/s/z64on3ypm",
+    description:
+      "react hook for hook that handles checking if addEventListener is supported, adding the event listener, and removal on cleanup",
```
Comment (itaditya) [inline hooks.js]:
```suggestion
      "react hook that checks if addEventListener is supported, add the event listener, and removal it on cleanup",
```

**Comment by itaditya** [comment]
Squash the commits into one and I'll merge ASAP as everything looks good to me

**Comment by anikethsaha** [comment]
WAT...how did it closed. I was rebasing and idk how it got closed while rebasing !.
@itaditya is it possible for you to re open it

**Comment by itaditya** [comment]
Strange, even I can't repoen the Pull Request. Can you a new PR with a single commit?

**Pull request closed without merging**